extensions/frontend/pkg/config/config.go

```diff
@@ -17,8 +17,9 @@ type Config struct {
 	GatewayEndpoint       string
 	SkipUserGroupsInToken bool
 
-	EnableFavorites          bool `yaml:"favorites"`
-	EnableProjectSpaces      bool
+	EnableFavorites          bool   `yaml:"favorites"`
+	EnableProjectSpaces      bool   `yaml:"project_spaces" env:"OCIS_PROJECT_SPACES_ENABLED;FRONTEND_PROJECT_SPACES_ENABLED" desc:"Indicates to clients that project spaces are supposed to be made available."`
+	EnableShareJail          bool   `yaml:"share_jail" env:"OCIS_SHARE_JAIL_ENABLED;FRONTEND_SHARE_JAIL_ENABLED" desc:"Indicates to clients that the share jail is supposed to be used."`
```

Comment:
are the two env vars used in more than one oCIS service? if not, a single, specific env var is good enough IMO.

Reply:
cool, reduced to the ones prefixed with `FRONTEND_`. thank you! 😇